app/routes/docs.terms-of-service.md

```diff
@@ -0,0 +1,38 @@
+---
+handle:
+  breadcrumb: Terms of Service
+---
+
+# Terms of Service
```

Comment:
Should we call this "Terms of Service" or "Code of Conduct"? "Terms of Service" suggests a legal enforcement mechanism.

Reply:
Yeah, I've wondered about the name too.  It could be a "Code of Conduct", but it's also a policies.  So maybe "Terms of Usage" or "GCN Usage Policies".

Reply:
A "Code of Conduct" can include policies for use too. I don't think I've ever heard of a document called "Usage Policies" but I know what a "Code of Conduct" is.